drivers/platform/nintendo3ds/ctr_pxi.c

```diff
@@ -477,7 +480,7 @@ static void vpxi_irq_worker(struct work_struct *work)
 		if (pending_mask & BIT(qirq)) {
 			unsigned long flags;
 			spin_lock_irqsave(&vpd->lock, flags);
-			list_for_each_entry(info, &vpd->vqs, node)
+			list_for_each_entry (info, &vpd->vqs, node)
```

Comment:
this added whitespace doesn't look right.

Reply:
doesn't look right to me either but this is what clang-format spits out with the default Linux config